source/assets/components/RecipeCardFea.js

```diff
@@ -0,0 +1,121 @@
+const $ = (selector) => document.querySelector(selector);
+
+class RecipeCardFea extends HTMLElement {
+  constructor() {
+    super();
+    this.shadow = this.attachShadow({ mode: 'open' });
+  }
+
+  set data(data) {
+    const styleElem = document.createElement('style');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group16/cse110-fa21-group16/pullRequest?prid=8393580)
```suggestion
    const styleElem = document.createElement("style");
```